src/pages/docs/getting-started/best-practices/installation-guidelines.mdx

```diff
@@ -178,7 +178,11 @@ If you plan on having external [polling Tentacles](/docs/infrastructure/deployme
 
 ### File Storage
 
-Octopus Deploy stores BLOB items such as task logs (generated during deployments), deployment artifacts, packages, project images on a file share instead of in the database.
+Octopus Deploy stores BLOB items such as task logs (generated during deployments), deployment artifacts, packages, project images, event exports on a file share instead of in the database.
+
+:::div{.hint}
```

Comment:
I don't think this hint block is needed here

Reply:
Deleted